### PR TITLE
#165305710 Redux Setup

### PR DESCRIPTION
**Pull request by Ogutu-Brian**
### What does this PR do?
Creates an appropriate folder/file structure for React/Redux workflow
### What are the tasks to be completed?
- Setup redux folder structure
- Install redux, react-redux and required dependencies
### How can this be manually tested?
After cloning the repository:
```
$ git checkout ch-redux-setup-165305710
$ npm install
$ npm run start:dev
```
### Screenshots
image

#### What are the relevant pivotal tracker stories?
[#165305710](https://www.pivotaltracker.com/story/show/165305710)

### CODE REVIEW

**Review by IssaIan — APPROVED** [review]

**Review by Bakley — APPROVED** [review]

**Review by a-braham — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -34,10 +34,15 @@
   "dependencies": {
     "@babel/core": "^7.4.5",
     "@babel/polyfill": "^7.4.4",
+    "eslint-plugin-react": "^7.13.0",
```
Comment (Bakley) [inline package.json]: @Ogutu-Brian The package` eslint-plugin-react` is included as both a dev and production dependency. Is that okay?

Comment (Bakley) [reply package.json]: line 32 and 37

Comment (Ogutu-Brian) [reply package.json]: Hello @Bakley ,
It should be a dev dependency. Let me fix that

**Comment by koechkevin** [comment]
Looks good to me

**Pull request merged**